Library/Formula/racer.rb

```diff
@@ -0,0 +1,20 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+
+  keg_only "You should set path in your special editor which use racer"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.mkpath
+    bin.install "target/release/racer"
+  end
+
+  test do
+    assert_match "RUST_SRC_PATH", `#{bin}/racer complete std::io::B`
```

Comment:
Use `shell_output("#{bin}/racer complete std::io::B")` instead of `` `#{bin}/racer complete std::io::B` ``.